indico/indico
# Add a new form to capture rejection reason for registration(s)

Test Instructions
- Create an event with enabled moderation in the registration form.
- Register for the event
- Go to the registration display page, reject the participant, a form in the popup window will appear to enter a reason for rejection and a switch to enable or disable attaching the reason with the notification.
- The reason will be stored in the database as `registration.rejection_reason`, added as data into the event log,
- Also available to use in email templates as `rejection_reason` tag.
- Similarly, while rejecting multiple registrations from the registration list the same popup window would appear.

> Squash and Merge pl.

- Let me know if there are any changes required.
